aocharmovie.cpp

```diff
@@ -3,6 +3,8 @@
 #include "misc_functions.h"
 #include "file_functions.h"
 #include "aoapplication.h"
+#include "debug_functions.h"
+#include <string>
```

Comment:
What's the point of this `<string>` include? Doesn't look like it's ever used.

Reply:
Leftover, same w/ the debug functions call. Will remove.